modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java

```diff
@@ -348,6 +348,10 @@ else if (cmd.equals(Constants.PREVIEW)) {
 
 			String portletResource = ParamUtil.getString(
 				actionRequest, "portletResource");
+			if (Validator.isNotNull(portletResource) ||
```

Comment:
Hi @kristofloneck 

The code does not compile, can you recheck it

Thanks!

Reply:
check it, fixed it.